modules/compiler/tools/muxc/CMakeLists.txt

```diff
@@ -28,11 +28,10 @@ target_compile_definitions(muxc PRIVATE
   muxc_CL_STD_30=$<BOOL:${muxc_CL_STD_30}>)
 target_include_directories(muxc SYSTEM PUBLIC ${LLVM_INCLUDE_DIR})
 target_include_directories(muxc PUBLIC
-  $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/modules/compiler/source/base/include>
   $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/source/cl/source/compiler/include>
   $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/modules/compiler/multi_llvm/include>)
 target_link_libraries(muxc PRIVATE
-  compiler-static mux builtins cargo LLVMBitWriter LLVMIRReader)
+  compiler-static compiler-base mux builtins cargo LLVMBitWriter LLVMIRReader)
```

Comment:
does it not get builtins and cargo as well from compiler-base?

Reply:
Yep, good point - and `mux`.